README.md

```diff
@@ -0,0 +1,53 @@
+## orbit-db-cache
+
+Isomorphic cache used by `orbit-db`, implemented using `level.js` (indexedDB) for the browser and `leveldown` (leveldb) for node.
+
+### Used in
+
+* [orbit-db](https://github/com/orbitdb/orbit-db)
+
+### API
+
+#### Public methods
+
+##### `open`
+
+Open cache store and make it available to be used (indexedDB or leveldb)
+
+##### `close`
+
+Close cache store and delete cached items
+
+##### `destroy`
+
+Completely remove an existing store
```

Comment:
Calling `destroy()` deletes the local leveldb too. That is, it can't be re-opened locally after calling destroy. We should write something like "...and deletes the locally persisted cache".

Reply:
Done!